src/server/api/metadata/index.ts

```diff
@@ -10,7 +10,7 @@ import { getSectionsMetadata } from './getSectionsMetadata'
 
 export const MetadataApi = {
   init: (express: Express) => {
-    express.get(ApiEndPoint.MetaData.metaCache(), AuthMiddleware.requireView, getMetaCache)
+    express.get(ApiEndPoint.MetaData.metaCache(), getMetaCache)
```

Comment:
This auth is removed:
When accessing table 1a, we call previous cycles meta cache
This fails, if the user has no role in previous cycle

Should be ok